docs/guides/platform/disk-images/managing-the-kernel-on-a-linode/index.md

```diff
@@ -0,0 +1,210 @@
+enable_h1: true
+---
+
+The primary component of every Linux system is the *Linux kernel*. The kernel interfaces with the system's hardware and it controls the operating system's core functionality. This guide covers the types of kernels that can be assigned to a Linode and how to view, update, or change the kernel.
+
+## Types of Kernels
+
+Your Linode is capable of running one of three kinds of kernels:
+
+-   **Upstream kernel** (or *distribution-supplied kernel*): This kernel is maintained and provided by your Linux distribution. A major benefit of this kernel is that the distribution was designed with this kernel in mind and all updates are managed through the distributions package management system. It also may support features not present in the Linode kernel (for example, [SELinux](/docs/security/getting-started-with-selinux/)).
+
+-   **Linode kernel:** Linode also maintains kernels that can be used on a Linode. If selected, these kernels are provided to your Linode at boot (not directly installed on your system). The [Current Kernels](https://www.linode.com/kernels) page displays a list of all the available Linode kernels.
+
+-   **Custom-compiled kernel:** A kernel that you compile from source. Compiling a kernel can let you use features not available in the upstream or Linode kernels, but it takes longer to compile the kernel from source than to download it from your package manager. For more information on custom compiled kernels, review our guides for [Debian, Ubuntu,](/docs/tools-reference/custom-kernels-distros/custom-compiled-kernel-debian-ubuntu/) and [CentOS](/docs/tools-reference/custom-kernels-distros/custom-compiled-kernel-centos-7/).
+
+Most of the distribution images available on Linode use the upstream distribution-supplied kernel by default.
+
+## Determining Which Kernel Is Installed
+
+There are may ways you can determine which kernel version is installed on your Linux system. The following instructions cover the most common methods:
+
+1.  Log in to the Linode through either [SSH](/docs/guides/connect-to-server-over-ssh/) or [Lish](/docs/guides/using-the-linode-shell-lish/).
+
+1.  Run one of the following commands to display the kernel version:
+
+    - `uname -r`: Outputs a single line with just the kernel version.
+    - `hostnamectl`: Outputs an organized list of system details, including the kernel version. Can only be used on distributions with [systemd](https://en.wikipedia.org/wiki/Systemd) (Debian, Ubuntu, CentOS, openSUSE, and more).
+    - `cat /proc/version`: Outputs a string that contains the kernel version and release details.
+
+The kernel version should be a string that's similar to `5.4.0-80-generic`. In this example, the kernel ends with `generic`, which is common in Ubuntu systems to designate that this is an upstream kernel. If the kernel ends with something similar to `linode123`, a Linode kernel is being used and the string exactly which Linode kernel.
```

Comment:
...a Linode kernel is being used and the string exactly *matches* which Linode kernel.